src/ntfs/mod.rs

```diff
@@ -242,23 +251,53 @@ mod tests {
     #[test]
     fn feed_infos_serialization_deserialization() {
         let mut feed_infos = BTreeMap::default();
-        feed_infos.insert("feed_license".to_string(), "".to_string());
-        feed_infos.insert("ntfs_version".to_string(), "0.3".to_string());
-        feed_infos.insert("feed_creation_date".to_string(), "20181004".to_string());
+        feed_infos.insert("tartare_platform".to_string(), "dev".to_string());
+        feed_infos.insert("feed_publisher_name".to_string(), "Nicaragua".to_string());
         let mut collections = Collections::default();
+        test_in_tmp_dir(|path| {
+            write::write_feed_infos(
+                path,
+                &feed_infos,
+                &collections.datasets,
+                get_test_datetime(),
+            )
+            .unwrap();
+            read::manage_feed_infos(&mut collections, path).unwrap();
+            let c_feed_infos: BTreeSet<_> = collections.feed_infos.iter().collect();
+            let expected: BTreeSet<_> = feed_infos.iter().collect();
+            let intersection: BTreeSet<_> = c_feed_infos.intersection(&expected).collect();
+            assert_eq!(intersection, expected.iter().collect());
+        });
+    }
 
+    #[test]
+    fn feed_infos_minimal() {
+        let mut collections = Collections::default();
+        collections.datasets = CollectionWithId::new(vec![Dataset::default()]).unwrap();
         test_in_tmp_dir(|path| {
-            write::write_feed_infos(path, &feed_infos).unwrap();
+            write::write_feed_infos(
+                path,
+                &collections.feed_infos,
+                &collections.datasets,
+                get_test_datetime(),
+            )
+            .unwrap();
             read::manage_feed_infos(&mut collections, path).unwrap();
-            let info_params: Vec<_> = collections.feed_infos.keys().collect();
-            // test that feed infos are ordered by info_param
-            assert_eq!(
-                info_params,
-                ["feed_creation_date", "feed_license", "ntfs_version"]
+            let dataset = &collections.datasets.values().next().unwrap();
+            let mut expected = BTreeMap::default();
+            expected.insert("feed_creation_date".to_string(), "20190403".to_string());
+            expected.insert("feed_creation_time".to_string(), "17:19:00".to_string());
+            expected.insert(
+                "feed_end_date".to_string(),
+                dataset.end_date.format("%Y%m%d").to_string(),
+            );
+            expected.insert(
+                "feed_start_date".to_string(),
+                dataset.start_date.format("%Y%m%d").to_string(),
             );
+            expected.insert("ntfs_version".to_string(), "0.7".to_string());
```

Comment:
u can use NTFS_VERSION if u don't want to change it later

Reply:
It was in debate. @datanel prefers to keep it fixed